### ISSUE TITLE
Add a preset for `barrier=sliding_gate`

### ISSUE THREAD

**Issue by Dimitar5555** [openstreetmap/id-tagging-schema#1171]
### OSM Tag(s)

barrier=sliding_gate

### How would you like this tag to see supported?

As a Preset

### Label

Sliding Gate

### Link to OSM Wiki page

https://wiki.openstreetmap.org/wiki/Tag:barrier%3Dsliding_gate

### Status of the Tag

In Use

### Usage of the tag

15 668

**Comment by Woazboat** [comment]
I don't think this tag should be promoted. The standard `barrier=gate` + `gate:type=*` should be used instead.
